internal/store/installation.go

```diff
@@ -197,15 +197,15 @@ func (sqlStore *SQLStore) CreateInstallation(installation *model.Installation) e
 		SetMap(map[string]interface{}{
 			"ID":               installation.ID,
 			"OwnerID":          installation.OwnerID,
+			"GroupID":          installation.GroupID,
+			"GroupSequence":    nil,
```

Comment:
Just out of curiosity, for the sake of style could we start sorting these sorts of object definitions alphabetically unless there's a more obvious sorting order, just so that changes like this are less likely to occur (where some lines just move for no real reason)?

Reply:
So, I did move that because I thought it made sense to have the various "ID"s in one spot near the top. That at least made it easier to read the JSON when I am listing installations so I figured we would match it here. Totally 0/5 on alphabetical or other ordering if that is what we want to do for store calls, but I do think trying to "logically" display the JSON payloads is helpful.